nixos/modules/image/repart-image.nix

```diff
@@ -89,11 +90,13 @@ let
   compressionPkg = {
     "zstd" = zstd;
     "xz" = xz;
+    "zstd-seekable" = zeekstd;
```

Comment:
As seekable zstd archives are backwards compatible, we should think about just generating them by default in the future instead of a separate option. However, for now, it's the sensible thing to have a separate option.

Reply:
I agree, the only downside would be a few kib of extra metadata, but that should be negligible compared to the size of an image.